NuGet.config

```diff
@@ -2,7 +2,8 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
+    <add key="AspNetCoreRelease" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />    
```

Comment:
Otherwise we get errors for 
Master - https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json
Microsoft.Extensions.ClosedGenericMatcher.Sources
Microsoft.Extensions.ObjectMethodExecutor.Sources
Microsoft.Extensions.SecurityHelper.Sources


Release - https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json
Missing Packages
System.Threading.Tasks.Channels.
System.IO.Pipelines.Extensions
System.Text.Formatting.
System.Threading.Tasks.Channels
System.Buffers.Primitives
System.Binary
System.Collections.Sequences.
System.Slices
System.Buffers.Primitives